src/main/resources/application-dev.yaml

```diff
@@ -82,4 +82,6 @@ logging:
 reminder:
   trigger-duration-days: "PT2M"
   cron-schedule-expression: "0 */5 * ? * *"
-
+synchronization:
+  frequency: "0 0 0 * * ?"
+  discoveryServiceId: "1"
```

Comment:
why not put it in the application.yml? If it is the same everywhere

Reply:
Yes I could; should I keep it only there and in the test one?

Reply:
if it is same everywhere then just put it in the application.yml